homeassistant/components/somfy_mylink/cover.py

```diff
@@ -65,6 +65,9 @@ class SomfyShade(RestoreEntity, CoverEntity):
     _attr_should_poll = False
     _attr_assumed_state = True
 
+    # Assumed state, by default we don't know if the shade is closed or not
```

Comment:
Please remove the comment. It is mostly helpfull now, so fits better in the commit message.